dataGeneration/generate-cosine-data-multi-entity.py

```diff
@@ -0,0 +1,296 @@
+'''
+
+Python script for ingesting sample data into opensearch index
+
+'''
+
+#{"index": {"_index":"host-cloudwatch","_id":"1177"}}
+#{"@timestamp":"2017-03-23T13:00:00","cpu":20.3, "memory":13,"host":"host1", "service": "service1"}
+
+import numpy as np
+from scipy.stats import uniform
+import datetime
+import time
+import random
+from random import Random
+from retry import retry
+import urllib3
+import concurrent.futures
+import argparse
+
+from opensearchpy import OpenSearch, RequestsHttpConnection
+from opensearchpy import helpers
+
+# https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
```

Comment:
Could you add docs on how to use the script in typical scenario?

Reply:
added readme